editor/gui/scene_tree_editor.h

```diff
@@ -58,6 +59,15 @@ class SceneTreeEditor : public Control {
 		BUTTON_UNIQUE = 9,
 	};
 
+	struct cached_node {
```

Comment:
```suggestion
	struct CachedNode {
```
For consistency with usual class/stuct naming.